libsolidity/ast/ASTJsonImporter.cpp

```diff
@@ -436,13 +437,13 @@ ASTPointer<UsingForDirective> ASTJsonImporter::createUsingForDirective(Json::Val
 		_node,
 		std::move(functions),
 		std::move(operators),
-		!_node.isMember("libraryName"),
-		_node["typeName"].isNull() ? nullptr  : convertJsonToASTNode<TypeName>(_node["typeName"]),
+		!_node.contains("libraryName"),
+		_node.contains("typeName") ? (_node["typeName"].is_null() ? nullptr  : convertJsonToASTNode<TypeName>(_node["typeName"])) : nullptr,
```

Comment:
```suggestion
		(_node.contains("typeName") && !_node["typeName"].is_null()) ? convertJsonToASTNode<TypeName>(_node["typeName"])) : nullptr,
```
again I think this is less of a headache to read

Reply:
There's a few more of these below, I won't comment on all of them :-).